src/java.desktop/share/classes/javax/swing/plaf/synth/SynthRootPaneUI.java

```diff
@@ -74,6 +76,10 @@ protected void uninstallDefaults(JRootPane root) {
 
         style.uninstallDefaults(context);
         style = null;
+        if (UIManager.getBoolean("RootPane.altPress")) {
+            KeyboardFocusManager.getCurrentKeyboardFocusManager().
+                    removeKeyEventPostProcessor(altProcessor);
+        }
```

Comment:
I think you have to keep track of whether `altProcessor` was installed or not. If the value of `"RootPane.altPress"` changes, the listener may be left installed.

Reply:
> I think you have to keep track of whether `altProcessor` was installed or not. If the value of `"RootPane.altPress"` changes, the listener may be left installed.

Yeah, if the value of "RootPane.altPress" is set to false, the listener won't be removed. Will update.